lib/robots/dor_repo/gis_assembly/generate_mods.rb

```diff
@@ -33,12 +33,34 @@ def purl
           @purl ||= Settings.purl.url + "/#{bare_druid}"
         end
 
-        def shp_file
-          @shp_file ||= Dir.glob("#{rootdir}/temp/*.shp").first
+        def vector_file
```

Comment:
Since we mostly distinguish between `raster` (ArcGis and GeoTIFF) and `vector` (Shapefile and GeoJSON) it made sense (in this particular case) to replace shapefile specific names with vector names